<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersWidget</class>
 <widget class="QWidget" name="UsersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户管理</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableView" name="usersTableView"/>
   </item>
   <item row="0" column="1">
    <widget class="QTableView" name="permissionsTableView"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>删除用户</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>创建用户</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
